products/PDFmorph.rst

```diff
@@ -0,0 +1,117 @@
+.. include:: ../abbreviations.txt
+
```

Comment:
@sbillinge [this](https://github.com/RomanMatthew/diffpy.github.io/blob/pdfmorph_web/products/PDFmorph.rst) would be the closest we can get (for now) to a real preview of what it'll look like on the page (some of the formatting and text that actually gets shown is different

Reply:
I just published the preview at http://pavoljuhas.github.io/diffpy.github.io.
There is currently no link from the menu to the PDFmorph page, which is at http://pavoljuhas.github.io/diffpy.github.io/products/PDFmorph.html.

@RomanMatthew - to create your own preview, please do
```sh
# ONCE
git fetch https://github.com/pavoljuhas/diffpy.github.io.git gh-pages:gh-pages
git push -u https://github.com/RomanMatthew/diffpy.github.io.git gh-pages:gh-pages

# to update the page
make html
make PUBLISHBRANCH=gh-pages publish-prepare publish-push
```

Reply:
@pavoljuhas Sorry if these questions are elementary, but how do we put the link to PDFmorph page into the products menu? I updated the index in this PR and thought that would do it - I'm a little new to the structure of this stuff right now. 

Getting on the sphinx stuff now!

Reply:
> ... how do we put the link to PDFmorph page into the products menu?

No problem - you need to add a new entry for PDFmorph to [_templates/menu01.html](https://github.com/diffpy/diffpy.github.io/blob/4f32f41ce1bc784daf165b70d69ed277c8d5675c/_templates/menu01.html#L16).
I think the reason why is this maintained manually is that we wanted to choose what shows in the product menu rather than have it auto-generated from sphinx titles.

PS2: A good way of checking how are other product pages put together is to use `git grep` -
```sh
git grep -liw pdfgui
```

Reply:
that particular fix (with the products menu) should be taken care of @pavoljuhas ! Thank you for the hosting tip!

Reply:
both of the last commands seem to have given me some issues - the last one gives 

`fatal: 'gh-pages' does not appear to be a git repository
fatal: Could not read from remote repository.`

Could the problem with the license be related to the path that I used? I'm not too sure how the path tracing works in .rst!

Reply:
> both of the last commands seem to have given me some issues - the last one gives
> fatal: 'gh-pages' does not appear to be a git repository fatal: Could not read from remote repository.

@RomanMatthew - I think it is easier to fix this over hangouts call, feel free to call me at my gmail.